src/services/clients/paymentApi.client.ts

```diff
@@ -0,0 +1,43 @@
+import 'reflect-metadata'
+
+import { AxiosInstance, AxiosRequestConfig, AxiosResponse } from 'axios'
+import { inject } from 'inversify'
+import { provide } from 'inversify-binding-decorators'
+
+import PaymentPostRequest from 'app/models/dto/paymentPostRequest'
+import PaymentPostResponse from 'app/models/dto/paymentPostResponse'
+import TYPES from 'app/types'
+
+@provide(PaymentApiClient)
+export class PaymentApiClient {
+  private readonly PAYMENT_API_ENDPOINT: string = `${this.PAYMENTS_API_URL}/payments`
```

Comment:
yeah we should be using the ch-sdk-node library for interacting with the payments api, rather than building our own client